nautobot/utilities/testing/api.py

```diff
@@ -256,6 +256,32 @@ def test_options_returns_expected_choices(self):
 
             self.assertEqual(set(self.choices_fields), field_choices)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_status_options_returns_expected_choices(self):
+            # Set to self.choices_fields as empty set to compare classes that shouldn't have any choice fields on serializer.
+            if not self.choices_fields:
+                self.choices_fields = set()
+
+            # Don't bother testing if there's no `status` field.
+            if "status" not in self.choices_fields:
+                self.skipTest("Object does not contain a `status` field.")
+
+            # Save self.user as superuser to be able to view available choices on list views.
+            self.user.is_superuser = True
+            self.user.save()
+
+            response = self.client.options(self._get_list_url(), **self.header)
+            actions = response.json()["actions"]["POST"]
+            choices = actions["status"]["choices"]
+
+            # Import Status here to avoid circular import issues w/ test utilities.
+            from nautobot.extras.models import Status  # noqa
+
+            # Assert that the expected Status objects matches what is emitted.
+            statuses = Status.objects.get_for_model(self.model)
+            expected = [{"value": v, "display": d} for (v, d) in statuses.values_list("slug", "name")]
+            self.assertListEqual(choices, expected)
```

Comment:
Do we need any sort of sorting on either or both of `choices` or `expected` to get consistent test results?

Reply:
I would say no, because both the choices and the expected list I constructed in the test were derived from a `Status` queryset that defaults to use `ordering = ["name"]` as defined in `Status.Meta.ordering`.